deployment/deployer.yaml

```diff
@@ -148,4 +172,51 @@ parameters:
   description: "Timeout for *expected* ops nodes to be present when cluster is recovering from a full restart."
   name: ES_OPS_RECOVER_AFTER_TIME
   value: "5m"
-
+-
+  description: "Node selector key for Elasticsearch cluster (key: value)."
+  name: ES_NS_KEY
```

Comment:
why not just use ES_NODESELECTOR (BTW NS = namespace) and not have separate KEY/VALUE?

    ES_NODESELECTOR=type=infra

Either way you can't specify multiple labels, so we don't get away from documenting the oc edit approach.